bigquery_storage/google/cloud/bigquery_storage_v1beta1/reader.py

```diff
@@ -27,16 +27,27 @@
     import pandas
 except ImportError:  # pragma: NO COVER
     pandas = None
+try:
+    import pyarrow
+except ImportError:  # pragma: NO COVER
+    pyarrow = None
 import six
 
+try:
+    import pyarrow
+except ImportError:  # pragma: NO COVER
+    pyarrow = None
+
 from google.cloud.bigquery_storage_v1beta1 import types
 
 
 _STREAM_RESUMPTION_EXCEPTIONS = (google.api_core.exceptions.ServiceUnavailable,)
-_FASTAVRO_REQUIRED = (
-    "fastavro is required to parse ReadRowResponse messages with Avro bytes."
-)
+
+_AVRO_BYTES_OPERATION = "parse ReadRowResponse messages with Avro bytes"
+_ARROW_BYTES_OPERATION = "parse ReadRowResponse messages with Arrow bytes"
+_FASTAVRO_REQUIRED = "fastavro is required to {operation}."
```

Comment:
I assume these parameterized errors are for when you do things like to_arrow with avro bytes?

Reply:
Yeah, this is vestigal from when I was planning to implement `to_arrow` for Avro streams. Removed for now.